main/libsodium/APKBUILD

```
@@ -22,13 +20,18 @@ build() {
 }
 
 check() {
-	cd "$builddir"
 	make check
 }
 
+static() {
+	pkgdesc="libsodium static libraries"
+
+	mkdir -p "$pkgdir"-static/usr/lib
```

Use `$subpkgdir` instead.
```suggestion
	mkdir -p "$subpkgdir"/usr/lib
```